website/docs/add-secure-apps/providers/ssf/index.md

```diff
@@ -1,30 +1,39 @@
+Shared Signals Framework (<abbr>SSF</abbr>) is a collection of standards for sharing asynchronous security signals and events across multiple applications and an identity provider. SSF leverages the APIs of the application and the IdP, using privacy-protected, secure webhooks.
 
-## About Shared Signals Framework
+## What are security signals?
 
-In authentik, an SSF provider allows applications to subscribe to certain types of security signals (which are then translated into SETs, or Security Event Tokens) that are captured by authentik (the IdP), and then the application can respond to each event. In this scenario, authentik acts as the _transmitter_ and the application acts as the _receiver_ of the events.
+An SSF provider allows applications to subscribe to _security signals_, translating them into Security Event Tokens (<abbr>SET</abbr>) that are captured by the IdP. In this arrangement, authentik acts as an mediator from an external event _transmitter_, ultimately with a specific application acting as the _receiver_ of the events.
 
-Events in authentik that are tracked via SSF include when an MFA device is added or removed, logouts, sessions being revoked by Admin or user clicking logout, or credentials changed.
+### Transmitter and receiver
+
+The role of the transmitter is to send signals to the receiver, and the receiver is the application that listens for the signals. Each system receives and sends signals, but the signals are different.
+
+The identity provider is the transmitter of _identity signals_ from the IdP to the application,
+and the application is the receiver of _identity signals_, such as user information, groups, and roles.
+
+An SSF provider is the transmitter of _security signals_ from the IdP to the application,
+and the application is the receiver of _security signals_, such as device information, user events, and security events.
+
+This shared signals approach allows for more dynamic security decisions than traditional static policies, as access determinations can adapt based on both identity context and device posture in near real-time.
 
 ## Example use cases
 
 A common use case for SSF is when an Admin wants to know if a user logs out of authentik, so that the user is then also automaticlaly logged out of all other work-focused applications.
 
-Another example use case is when an application uses SSF to subscribe to authorization events because the application needs to know if a user changed their password in authentik. If a user did change their password, then the application receives a POST request to write the fact that the password was changed.
-
-## About using SSF in authentik
+Another example use case is when an application uses SSF to subscribe to authorization events because the application needs to know if a user changed their password in authentik. If a user changes their password, then the application will receive a POST request to relay an event that the password was changed.
```

Comment:
Is event is for sure the correct term here...? I want to be careful with our use of the word event (versus signal, and also versus our other types of events).